examples/README.md

```diff
@@ -0,0 +1,25 @@
+# AWX Client Examples
```

Comment:
Having this in a separate file made sense in the original OpenShift auto-heal project, but here it may be better to have this information directly inside the top level `README.md` file.

Reply:
I plan to add usage examples in the root README for all the public interfaces. What i'd like to have eventually is individual code snippets for each  interface item in the root README and real life examples in the examples dir: client creation and going through an entire flow that makes sense for a user.

I think i'd prefer the instruction for running examples in the examples dir, but i'm not locked on the idea.